src/scripts/main.js

```
@@ -354,7 +354,49 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+if (!table) {
+  throw new Error('Dashboard is not found');
+}
```

Good: Selecting the table with document.querySelector('.dashboard') and throwing an error if not found is correct and satisfies requirement 2.4. Keep this defensive check in place so the script fails fast if the expected DOM node is absent.